packages/admin/resources/views/resources/pages/edit-record.blade.php

```diff
@@ -6,18 +6,42 @@
         'filament-resources-record-' . $record->getKey(),
     ])"
 >
-    <x-filament::form wire:submit.prevent="save">
-        {{ $this->form }}
+    @capture($form)
+        <x-filament::form wire:submit.prevent="save">
+            {{ $this->form }}
 
-        <x-filament::form.actions
-            :actions="$this->getCachedFormActions()"
-            :full-width="$this->hasFullWidthFormActions()"
-        />
-    </x-filament::form>
+            <x-filament::form.actions
+                :actions="$this->getCachedFormActions()"
+                :full-width="$this->hasFullWidthFormActions()"
+            />
+        </x-filament::form>
+    @endcapture
 
-    @if (count($relationManagers = $this->getRelationManagers()))
-        <x-filament::hr />
+    @php
+        $relationManagers = $this->getRelationManagers();
+    @endphp
 
-        <x-filament::resources.relation-managers :active-manager="$activeRelationManager" :managers="$relationManagers" :owner-record="$record" :page-class="static::class" />
+    @if ((! $this->hasCombinedRelationManagersWithForm()) || (! count($relationManagers)))
+        {{ $form() }}
```

Comment:
```suggestion
        {{ $form }}
```

Reply:
No, this is the syntax for the Blade @capture directive

Reply:
I'm sorry, I misread the package docs.